src/StoreKeeper/WooCommerce/B2C/PaymentGateway/PaymentGateway.php

```diff
@@ -581,4 +579,14 @@ public function addGatewayClasses($default_gateway_classes)
 
         return array_merge($default_gateway_classes, $gateway_classes);
     }
+
+    protected function storekeeperOrderHasRefundWithoutReturnPayment(array $order): bool
+    {
+        $hasRefund = false;
+        if (isset($order['paid_back_value_wt']) && 0 != $order['paid_back_value_wt'] && 0 != $order['refunded_price_wt']) {
```

Comment:
if paid_back_value_wt != 0 than it had payment, so function name is not correct : P